snapcraft/plugins/go.py

```diff
@@ -207,29 +266,32 @@ def clean_build(self):
         if os.path.isdir(self._gopath_pkg):
             shutil.rmtree(self._gopath_pkg)
 
-    def _run(self, cmd, **kwargs):
+    def _run(self, cmd: List[str], cwd: str = None, **kwargs) -> None:
```

Comment:
While not used here, we used this signature all over the place to avoid head scratches when wanting to pass additional arguments to Popen, we can reassess this if required.